CHANGELOG.next.asciidoc

```diff
@@ -424,6 +424,7 @@ otherwise no tag is added. {issue}42208[42208] {pull}42403[42403]
 - The journald input is now generally available. {pull}42107[42107]
 - Add metrics for number of events and pages published by HTTPJSON input. {issue}42340[42340] {pull}42442[42442]
 - Filestram input now supports migrating state when changing its ID, for that set `previous_ids`. {issue}42472[42472] {pull}42624[42624]
+- Add `etw` input fallback to attach an already existing session. {pull}42847[42847]
```

Comment:
Is it supposed to be here? It seems unrelated to this PR

Reply:
That came when I merged `main` onto my branch, I'll fix it. Thanks for spotting that!